docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express')
+const { CloudEvent, HTTPReceiver } = require('cloudevents-sdk')
+const HTTPBinary = require('cloudevents-sdk/lib/bindings/http/emitter_binary_1.js')
```

Comment:
@csantanapr I noticed that you're pointing to a commit hash in your dependency now. If you are interested in getting something even newer (hopefully a release 2.0 will be by end of month), you can do this for an emitter. Default is v1 binary.

```js
const { CloudEvent, HTTPEmitter } = require("cloudevents-sdk");

// With only an endpoint URL, this creates a v1 emitter
const v1Emitter = new HTTPEmitter({
  url: "https://cloudevents.io/example"
});
const event = new CloudEvent()
  .type(type)
  .source(source)
  .time(new Date())
  .data(data)

// By default, the emitter will send binary events
v1Emitter.send(event).then((response) => {
    // handle the response
  }).catch(console.error);
```

Reply:
I always have be pointing to a hash And being updated from day to day, I will update with latest changes. Thanks for the quick updates to the sdk and reviewing this PR